package/Dockerfile

```diff
@@ -29,7 +29,7 @@ RUN mkdir -p /ganesha-extra \
     && mkdir -p /ganesha-extra/etc/dbus-1/system.d \
     && cp src/scripts/ganeshactl/org.ganesha.nfsd.conf /ganesha-extra/etc/dbus-1/system.d/
 
-FROM registry.fedoraproject.org/fedora-minimal:30 AS run
+FROM registry.fedoraproject.org/fedora-minimal:30
```

Comment:
I am not sure if we should remove `AS run`? @joshimoo ?

Reply:
AFAIK, it's just a name for the base image in the multi-stage build process, and it is not referenced or used after that.

Reply:
Agree, removing it shouldn't cause any problem. Since we don't refer to the specific run stage in any of our scripts.
More information about the multistage builds here: https://docs.docker.com/develop/develop-images/multistage-build/#name-your-build-stages